src/bootstrap/src/core/config/config.rs

```diff
@@ -2884,6 +2880,20 @@ impl Config {
 
         Some(commit.to_string())
     }
+
+    /// Check for changes in specified directories since a given commit.
+    /// Returns true if changes exist, false if no changes
+    pub fn check_for_changes(&self, dirs: &[PathBuf], commit: &str) -> bool {
```

Comment:
I just found we already have very similar function to this:

https://github.com/rust-lang/rust/blob/df1b5d3cc2117f1ee96abca25678bc5f5604d450/src/bootstrap/src/core/config/config.rs#L2839-L2841

Can you utilize this function instead?

Reply:
I already fixed conflicts with rebase 2 days ago

Reply:
> I already fixed conflicts with rebase and force pushed 2 days ago.

I couldn't catch that because of "S-waiting-on-author" label.